### PR TITLE
Refactor & organise features/assessment folder

### PR DESCRIPTION
**Pull request by selvach**
# Description

- Fixes spelling of `version` in AppConfig.ts
- Moves AssessmentCoordinator to ./src/core/assesment
- Renames YourCovidTests -> CovidTests

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/core/assessment/AssessmentCoordinator.ts [hunk 1]
```diff
@@ -31,13 +30,13 @@ export class AssessmentCoordinator {
       if (this.assessmentData.currentPatient.isHealthWorker) {
         this.navigation.navigate('HealthWorkerExposure', { assessmentData: this.assessmentData });
       } else {
-        this.navigation.navigate('CovidTest', { assessmentData: this.assessmentData });
+        this.navigation.navigate('CovidTestList', { assessmentData: this.assessmentData });
```
Comment (cm-hvg) [inline src/core/assessment/AssessmentCoordinator.ts:33]: Just a thought, maybe we can have routes in enum in the future?

Comment (selvach) [reply src/core/assessment/AssessmentCoordinator.ts:33]: They are "typed"  using `keyof ScreenParamList`

Comment (selvach) [reply src/core/assessment/AssessmentCoordinator.ts:33]: Not sure if that's an Expo navigator style?

Comment (cm-hvg) [reply src/core/assessment/AssessmentCoordinator.ts:33]: Oh yeah, you are right. Is already typed. My bad, I thought it wasn't because it is a string.

**Pull request merged**